packages/fx-core/src/common/versionMetadata.ts

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export const MetadataV3 = {
+  projectVersion: "1.0.0",
+  platformVersion: {
+    vs: "17.5.x.x",
```

Comment:
is this pattern only work for 17.5 preview?

Reply:
The value is not used in vs 17.5 fallback api.
Will update the value in dev branch and remove it here.